tests/functional_tests/helper_scripts/scoreboard/scoreboard_test_teardown.sh

```diff
@@ -10,4 +10,5 @@ do
 done
 cd ../../main_server || exit
 node main_server.js >>/tmp/log/main_server.log 2>&1 &
+PROCESSES+=("$!")
```

Comment:
This script is invoked in line-31 of tests/functional_tests/autolab.sh file. I presume that the script is helping start the main server. A cleanup script does not typically start a new server.

It is better to push the command to the following location.
```bash
> node main_server.js >> /tmp/log/main_server.log 2>&1 &
> bats missing_files.bats
```

Reply:
Also the process number being collected would be killed at the end of the shell script. For the sake of readability (top-down rule), a variable declared in one shell script would be used mostly in the same script.

Obviously the environment variables are an exception to this rule. The environment variables come from the config file. For the rest of the variables, it is better to keep the shell variables in the same file.

Reply:
The reason for including this temporary change here is we have a pending PR which actually deals with correcting the testing structure. It is dealt with there. The need for ending those processes here was to allow codecov to run the tests without failure.

Reply:
I agree with the need. I am just hoping to shift the work from cleanup script to the top-level so that the intent is apparent.